test/BUILD.bazel

```diff
@@ -100,13 +100,15 @@ cc_test(
         "//eventuals",
         "//test/concurrent",
         "@com_github_google_googletest//:gtest_main",
-    ] + select({
-        # We support backward lib only for macOS and Linux.
-        # The support for Windows might be added in future.
-        # https://github.com/3rdparty/eventuals/issues/450
-        "@bazel_tools//src/conditions:windows": [],
-        "//conditions:default": [
-            "@com_github_3rdparty_bazel_rules_backward_cpp_stacktrace//:backward-stacktrace",
-        ],
-    }),
+    ] + select(
+        {
+            # We support backward lib only for Linux.
+            # The support for Windows and MacOS might be added in future.
+            # https://github.com/3rdparty/eventuals/issues/450
```

Comment:
This issue does not cover MacOS. You can edit the issue to also cover MacOS; please also describe why it currently doesn't work on MacOS.